### PR TITLE
rtt localization for East Siauliai Woods

### PR DESCRIPTION
**Pull request by terrapin47**

### CODE REVIEW

**Comment by Sunaries** [comment]
Good to merge.

File: ETC.tsv [hunk 1]
```diff
@@ -11012,7 +11012,7 @@ ETC_20150414_011013	Recipe - Katzbalger
 ETC_20150414_011014	Recipe - Dress Sword
 ETC_20150414_011015	{memo X}Recipe - Crude Cane Rover
 ETC_20150414_011016	Recipe - Aras' Long Rod
-ETC_20150414_011017	Recipe - Snake Rod
+ETC_20150414_011017	Recipe - Serpentine Rod
```
Comment (Sunaries) [inline ETC.tsv]: If you're changing the original from Snake to Serpentine, you have to let IMC know, because the wiki still says Snake.

**Comment by Sunaries** [comment]
Good to merge.

If you're changing the original from Snake Rod to Serpentine Rod, you have to let IMC know, because the wiki still says Snake.

**Comment by terrapin47** [comment]
Changed the item from 'Serpentine Rod' to 'Snake Rod' instead.

**Comment by Sunaries** [comment]
Good to merge. :+1:

**Comment by terrapin47** [comment]
Oops, fixed the outstanding conflicts, @imcgames

**Comment by imcgames** [comment]
:+1:

**Pull request merged**